src/constants.js

```diff
@@ -2,9 +2,9 @@
 
 const inputMapping = 'input-mapping.json'
 const inputValidation = 'input-validation.json'
-const meta = 'meta.json'
+const inputMeta = 'meta.json'
 
-exports.expectedEndpointsDirectories = [inputMapping, inputValidation, meta]
+exports.expectedEndpointsDirectories = [inputMapping, inputValidation, inputMeta]
 exports.inputMapping = inputMapping
 exports.inputValidation = inputValidation
-exports.meta = meta
+exports.inputMeta = inputMeta
```

Comment:
Minor: Lets maybe re-order these into a more logical setup flow. Meta needs to be interpreted first for setting up routes,validation needs to pass before mapping. E.G `[inputMeta, inputValidation, inputMapping]`